src/test/resources/pdf/testutil/stubs/hmpps-restricted-patients-api-stub.json

```diff
@@ -2,6 +2,7 @@
   "content": {
     "prisonerNumber": "A1234AA",
     "supportingPrisonDescription": "HMP Exeter",
```

Comment:
again would be good to remove use of description here so I can remove from restricted patients api

Reply:
Removed